controllers/article.js

```diff
@@ -0,0 +1,52 @@
+import 'babel-polyfill';
+import models from '../models';
+import { generateSlug } from '../helpers';
+
+const { Article, User, Profile } = models;
+
+export default {
+  createArticle: async (req, res, next) => {
+    const {
+      title, body, description, category,
+    } = req.body;
+
+    const userId = req.decoded.id;
+    const user = await User.findOne(
+      {
+        where: { id: userId },
+        include: [{ model: Profile }]
+      });
+    const articleExists = await Article.findOne(
+      { where: { title, userId } }
+    );
+
+    if (articleExists) {
+      const errorMessage = 'User already has an article with this title';
```

Comment:
The message here is best written as `You already have an article with the same title`

Also, this part should be a middleware. Remember we have only operations that are successful in our controllers.